tests/test_evidence_strings.py

```diff
@@ -244,6 +244,11 @@ def test_date(self):
         self.assertEqual(self.test_ges['evidence']['variant2disease']['date_asserted'], date_string)
         self.assertEqual(self.test_ges.date, date_string)
 
+    def test_validate(self):
+        test_args = get_args_CTTVGeneticsEvidenceString_init()
+        test_evidence_string = ES.CTTVGeneticsEvidenceString(*test_args)
+        test_evidence_string.validate()
```

Comment:
While checking the `validate` method to know how `jsonschema.validate` works, I saw the last line runs `self.disease.is_obsolete()`, but it doesn't return anything. Any idea why is that done?